README.md

```diff
@@ -46,38 +46,30 @@ Here's a table summarizing this:
+to examples that make use of them):
+
+- [Metamers](http://www.cns.nyu.edu/~lcv/texture/):
+  given a model and a reference image, stochastically generate a new image whose
+  model representation is identical to that of the reference image.
+- [Eigendistortions](https://www.cns.nyu.edu/~lcv/eigendistortions/):
+  given a model and a reference image, compute the image perturbation that produces
+  the smallest and largest changes in the model response space.  These correspond to the
+  minimal/maximal eigenvectors of the Fisher Information matrix of the representation (for deterministic models, 
+  the minimal/maximal singular vectors of the Jacobian).
 - [Maximal differentiation (MAD)
-  competition](https://www.cns.nyu.edu/pub/lcv/wang08-preprint.pdf):
-  given two models and an image, synthesize two pairs of images: two
-  that the first model represents identically, while the second model
-  represents as differently as possible; and two that the first model
-  represents as differently as possible while the second model represents
-  identically.
+  competition](https://ece.uwaterloo.ca/~z70wang/research/mad/):
+  given two models that measure distance between images and a reference image, generate pairs of 
+  images that optimally differentiate the models.  Specifically, synthesize a pair of images 
+  that the first model says are equi-distant from the reference while the second model says they 
+  are maximally/minimally distant from the reference. Synthesize a second pair with the roles of the two models reversed.
 - [Geodesics](https://www.cns.nyu.edu/pub/lcv/henaff16b-reprint.pdf):
-  given a model and two images, synthesize a sequence of images that form
-  a short path in the model's representation space. That is, a sequence of 
-  images that are represented to as close as possible to a straight interpolation 
-  line between the two anchor images.
-- [Eigendistortions](https://www.cns.nyu.edu/pub/lcv/berardino17c-final.pdf):
-  given a model and an image, synthesize the most and least noticeable
-  distortion on the image (with a constant mean-squared error in
-  pixels). That is, if you can change all pixel values by a total of
-  100, how does the model think you should do it to make it as obvious
-  as possible, and how does the model think you should do it to make
-  it unnoticeable.
+  given a model and two images, synthesize a sequence of images that lie on 
+  the shortest ("geodesic") path in the model's representation space. 
   
-(where for all of these, "identical (resp. different) representation",
-stands for small (resp. large) l2-distance in a model's representation space)
-
 # Status
 
-This project is currently in alpha, under heavy development. Not all features
-have been implemented, and there will be breaking changes.
+This project is currently under heavy development. Not all features
+have been implemented, and there will be significant changes.
```

Comment:
I'd like to keep the phrase "breaking changes", which is specific that we're not guaranteeing code written with plenoptic now might not work in several months. "significant changes" is also true, but feels less specific.

Reply:
agree -- changed